_drafts/rails-migrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Rails Migrations
+---
+
+We had a lively conversation today at [Big Nerd Ranch][bnr] about Rails migrations.
+What are they really for?
+How long should you keep them around?
+What about this crazy schema.rb file?!
+TIL WE HAZ OPINIONS!
+
+⚠️ The views of this post are my own.
+This is in no away a representation of our team's view of the subject.
+Quite the contrary, actually.
```

Comment:
You can sort of see by that that I expected to come to the conclusion "delete migrations", but that didn't happen. So I'd say we all agree more than not to the actual conclusion of the post. Maybe I'll soften that a bit?